sdk/appservice/mgmt/src/main/java/com/azure/management/appservice/implementation/WebAppMsiHandler.java

```diff
@@ -120,7 +120,7 @@ WebAppMsiHandler withExistingExternalManagedServiceIdentity(Identity identity) {
      * @return WebAppMsiHandler
      */
     WebAppMsiHandler withoutExternalManagedServiceIdentity(String identityId) {
-        this.userAssignedIdentities.put(identityId, null);
+        this.userAssignedIdentities.remove(identityId);
```

Comment:
@xccc-msft Please let me know why it putting `null` to map value.

This causes NPE during client side validation.

Reply:
Just update for tracking purpose: This is by design behavior as the server need to verify null as value in the map to understand it requests to remove the `identityId`.